offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/MaintainProductsController.groovy

```diff
@@ -68,6 +66,27 @@ class MaintainProductsController {
         }
     }
 
+    /**
+     * Triggers the copy Use Case of a product
+     *
+     * @param category The products category which determines what kind of product is created
+     * @param description The description of the product
+     * @param name The name of the product
+     * @param unitPrice The unit price of the product
+     * @param unit The unit in which the product is measured
+     */
+    void copyProduct(ProductCategory category, String description, String name, double unitPrice, ProductUnit unit, ProductId productId){
+        try{
+            println("You did it!")
+            //ToDo how should the Id be provided to the Use Case?
+            /**Product product = ProductConverter.createProduct(category, description, name, unitPrice, unit)
+            copyProductInput.copyModified(product) */
```

Comment:
why is that commented out? I think you need the call

Reply:
I think how this interface is handled doesn't make sense. We create a product from the selection in the view but we don't transfer the old productId here. How should we know what the original productId was to compare it later?